frontend/components/Landing/MeetupSpeakers.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="bg-white">
+    <div class="mx-auto max-w-7xl py-12 px-4 sm:px-6 lg:px-8 lg:py-24">
+      <div class="space-y-12 lg:grid lg:grid-cols-3 lg:gap-8 lg:space-y-0">
+        <div class="space-y-5 sm:space-y-4">
+          <h2 class="text-3xl font-bold tracking-tight sm:text-4xl">Prezentacje na najbliższym spotkaniu <br>
+            <span class="text-laravel">10 listopada 2022r.</span>
+          </h2>
+        </div>
+        <div class="lg:col-span-2">
+          <ul role="list" class="space-y-12 font-bold sm:grid sm:grid-cols-2 sm:gap-x-6 sm:gap-y-12 sm:space-y-0 lg:gap-x-8">
+            <li v-for="person in people" :key="person.name">
+              <div class="space-y-4">
+                <img class="mx-auto h-40 w-40 rounded-full xl:h-56 xl:w-56" :src="person.image" alt="" />
+                <div class="space-y-1 text-lg text-laravel  font-bold font-medium leading-6">
+                  <router-link :to="{ name: 'people' }"><h3>{{ person.name }}</h3></router-link>
+                  <router-link :to="{ name: 'companies' }"><p class="text-zinc-600">{{ person.company }}</p></router-link>
+                </div>
+                <div class="text-lg">
+                  <router-link :to="{ name: 'meetups' }"><p class="text-2xl bold">{{ person.bio }}</p></router-link>
+                </div>
+
+                <ul role="list" class="flex space-x-5">
+                  <li>
+                    <a :href="person.twitterUrl" class="text-gray-400 hover:text-gray-500">
+                      <span class="sr-only">Twitter</span>
+                      <svg class="h-5 w-5" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20">
+                        <path d="M6.29 18.251c7.547 0 11.675-6.253 11.675-11.675 0-.178 0-.355-.012-.53A8.348 8.348 0 0020 3.92a8.19 8.19 0 01-2.357.646 4.118 4.118 0 001.804-2.27 8.224 8.224 0 01-2.605.996 4.107 4.107 0 00-6.993 3.743 11.65 11.65 0 01-8.457-4.287 4.106 4.106 0 001.27 5.477A4.073 4.073 0 01.8 7.713v.052a4.105 4.105 0 003.292 4.022 4.095 4.095 0 01-1.853.07 4.108 4.108 0 003.834 2.85A8.233 8.233 0 010 16.407a11.616 11.616 0 006.29 1.84" />
+                      </svg>
```

Comment:
Maybe instead of svg we should create component for that icon/icons? We will have reusable components.